.storybook/preview.js

```diff
@@ -15,7 +14,5 @@ export const parameters = {
 			order: ["Introduction", "Form", ["Frontend Engine", "Validation Schema", "Conditional Rendering"]],
 		},
 	},
+	decorators: [withA11y],
```

Comment:
there's a warning `export 'withA11y' (imported as 'withA11y') was not found in '@storybook/addon-a11y'`

seems like registering the addon is sufficient

Reply:
I didn't get the warning actually but it appears there is no need to use the `withA11y` decorator indeed, I have removed the use of it